app/src/main/java/com/napzak/market/presentation/registration/RegistrationScreen.kt

```diff
@@ -0,0 +1,339 @@
+                text = uiState.description,
+                placeholder = stringResource(R.string.regi_description_placeholder),
+                onTextChange = onDescriptionChange,
+                isTitle = false,
+                maxLength = 240,
+            )
+        }
+        if (registrationType == TradeType.SELL) {
+            item {
+                Spacer(modifier = Modifier.height(35.dp))
+                RegistrationSellGroup(
+                    modifier = paddedModifier,
+                    salePrice = uiState.productSalePrice,
+                    salePricePlaceHolder = stringResource(R.string.regi_price_placeholder),
+                    onSalePriceChange = onSalePriceChange,
+                    productCondition = uiState.productCondition,
+                    onProductConditionChange = onProductConditionChange,
+                    postFeeType = if (uiState.isPostFeeIncluded) PostFeeType.INCLUDED else PostFeeType.EXCLUDED,
+                    onPostFeeChange = onPostFeeChange,
+                    isNormalPostChecked = uiState.isNormalPostChecked,
+                    onNormalPostCheckedChange = {
+                        onNormalPostStateChange(it)
+                        if (!uiState.isNormalPostChecked) onNormalPostFeeChange(BLANK)
+                    },
+                    normalPostFee = uiState.normalPostFee,
+                    onNormalPostFeeChange = onNormalPostFeeChange,
+                    isHalfPostChecked = uiState.isHalfPostChecked,
+                    onHalfPostCheckedChange = {
+                        onHalfPostStateChange(it)
+                        if (!uiState.isHalfPostChecked) onHalfPostFeeChange(BLANK)
+                    },
+                    halfPostFee = uiState.halfPostFee,
+                    onHalfPostFeeChange = onHalfPostFeeChange,
+                )
+            }
+        }
+        if (registrationType == TradeType.BUY) {
+            item {
+                Spacer(modifier = Modifier.height(35.dp))
+                RegistrationBuyGroup(
+                    modifier = paddedModifier,
+                    number = uiState.productPurchasePrice,
+                    onNumberChange = onPurchasePriceChange,
+                    isOfferAvailable = uiState.isOfferAvailable,
+                    onCheckChange = onOfferCheckChange,
+                )
+            }
+        }
+        item {
+            CommonButton(
+                modifier = paddedModifier
+                    .fillMaxWidth(),
+                text = stringResource(R.string.register),
+                onClick = { /*TODO*/ },
+                buttonColors = ButtonDefaults.buttonColors().copy(
+                    containerColor = NapzakMarketTheme.colors.purple30,
+                    contentColor = NapzakMarketTheme.colors.white,
+                    disabledContainerColor = NapzakMarketTheme.colors.gray400,
+                    disabledContentColor = NapzakMarketTheme.colors.white,
+                ),
```

Comment:
p2: 이렇게 바꾸시는건 어떠신가요?
```suggestion
                buttonColors = with(NapzakMarketTheme.colors) { 
                    ButtonDefaults.buttonColors().copy(
                       containerColor = purple30,
                       contentColor = white,
                       disabledContainerColor = gray400,
                       disabledContentColor = white,
                   ),
                }
```

Reply:
너무 좋은데요 ㅋ 바로 반영하겠습니다